docs/detections/alerts-view-details.asciidoc

```diff
@@ -22,29 +22,54 @@ image::images/alert-details-flyout.png[Alert details flyout]
 +
 Alert prevalence data can help you investigate relationships with other alerts and gain context about the events producing alerts. You can also click the alert prevalence count to explore the alerts in Timeline.
 +
 IMPORTANT: Before investigating alert prevalence data in Timeline, save any Timelines you're working on to ensure you can access them later.
 
-* *Enriched data*: Available threat indicator matches and threat intelligence data. This section only displays for alerts with intelligence data. Click the info icon to learn more about what data is collected. Refer to <<enriched-data-overview>> for more information.
+* *Insights*: Shows relationships with related alerts to help you quickly identify patterns. Refer to <<alert-details-insights, Insights on alerts>> for more information.
+
+* *Enriched data*: Displays available threat indicator matches and threat intelligence data. This section only displays for alerts with intelligence data. Click the info icon to learn more about what data is collected. Refer to <<enriched-data-overview>> for more information.
 +
 [role="screenshot"]
 image::images/enriched-data-info-icon.png[Informational message on enriched data, 600]
 
+[discrete]
+[[alert-details-insights]]
+==== Insights on alerts
+
+The Insights section shows you how an alert is related to other alerts and offers ways to investigate related alerts. You can use this information to quickly find patterns between alerts and then take action.
+
+Within the Insights section, you can click on the title for each insight to expand or collapse it. Some sections are only offered under a certain license and might require you to also enable a feature flag.
+
+[role="screenshot"]
+image::images/insights-section.png[Informational message on enriched data, 600]
+
+The Insights section provides the following details:
+
+* *Cases related to the alert* - Shows the total number and names of cases to which the alert has been added. Click a case's name to open its details.
+* *Alerts related by source event* - Shows the ten most recent alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the *Investigate in timeline* button to examine related alerts in Timeline.
+* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same Session ID, which is a unique ID for tracking a given Linux session.
++
+beta::[]
++
+NOTE: This feature requires a Platinum license. In addition, the *Include session data* setting must be enabled on your {endpoint-cloud-sec} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

Comment:
Does it require _only_ a Platinum license, or Platinum or higher? ([Some of our docs](https://www.elastic.co/guide/en/security/master/host-isolation-ov.html) use the phrase "Platinum or Enterprise" instead...?)

Reply:
Plat or higher sounds right - @michaelolo24 @janmonschke would you mind confirming. Thanks!

Reply:
Yep, plat or higher is correct :)